rust/crates/langsmith-pyo3/src/blocking_tracing_client.rs

```diff
@@ -47,8 +47,7 @@ impl BlockingTracingClient {
         Ok(Self { client: Arc::from(client) })
     }
 
-    // N.B.: We use `Py<Self>` so that we don't hold the GIL while running this method.
```

Comment:
The GIL info here was outdated, so I removed it. The current implementation drops the GIL explicitly by calling `allow_threads()`.